### DISCUSSION TITLE
WIX the system cannot find the file build vs 2019

### DISCUSSION THREAD

**Discussion by jlalanda** [wixtoolset/issues#6508] [Questions] [answered]
Hello,

I am having an error when the wix setup projet is compiling:

The error happens in the FileComponents wix and it returns the following error:

"The System cannot find the file 'directory..\Global.asax'

* Which version of WiX are you building with? 
Wix toolset v.3.11.2.4516

* Which version of Visual Studio are you building with (if any)? 
Visual studio 2019

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)? 
Wix Toolset Visual Studio Extension 1.0.0.4

* Which version of .NET are you building with? 
.NET Framework 4.5

* Describe the problem and the steps to reproduce it.
I am having a issue with the build of wix setup for a .NET Web Service.
Everything builds correctly expect the wix setup and it returns a error saying the file is not found but when i open the file path that the error returns the file exists and it opens succesufly.

* Describe the behavior you expected and how it differed from the actual behavior.
I am expecting that the project build the setup correctly

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions

**Reply by jlalanda** [reply]
hello,

@barnson but the issue will still be looked at, right?

**Comment by jlalanda** [comment, marked as answer]
I think this is the same problem:

https://github.com/wixtoolset/issues/issues/5314
